docs/en/token_endpoint.rst

```diff
@@ -301,6 +301,60 @@ The claims available in the *ID Token* are given below.
  - https://openid.net/specs/openid-connect-core-1_0.html#IDToken
  - https://openid.net/specs/openid-igov-openid-connect-1_0-03.html#Section-3.1
 
+Refresh Token
++++++++++++++
+
+The *Refresh Token* is a signed JWT that MAY be issued by the OP and MAY be used to obtain a new *Access Token* that enables the RP to access the UserInfo endpoint without direct user interaction. To obtain a *Refresh Token*, the RP MUST include in the scope parameter of the authentication request the value offline_access. The use of this scope can be useful in scenarios where an RP needs to verify that an end user's digital identity is still valid (e.g., it has not been revoked) or wants to keep attributes it has previously collected during the authentication phase up-to-date, such as for sending notifications to the end user after the end user has been authenticated.
+
+The *Refresh Token* MUST contain at least the following parameters.
+
+.. list-table:: 
+  :widths: 20 60 20
+  :header-rows: 1
+
+  * - **Claim**
+    - **Description**
+    - **Supported by**
+  * - **iss** 
```

Comment:
I suggest to use the same wording we have for these claims in the Federation specs, I mean for the english version

Reply:
what do you mean by "same wording"??

Reply:
instead of adopting new text/description of the single claim, we should use the existing text we already have in the international specs, at least for the english version